packages/auth/src/model/public_types.ts

```diff
@@ -170,12 +171,14 @@ export interface AuthSettings {
  * @public
  */
 export interface Auth {
-  /** The name of the app associated with the Auth service instance. */
+  /** The {@link @firebase/app#FirebaseApp} associated with the `Auth` service instance. */
+  readonly app: FirebaseApp;
+  /** The name of the app associated with the `Auth` service instance. */
   readonly name: string;
   /** The {@link Config} used to initialize this instance. */
   readonly config: Config;
   /**
-   * Changes the type of persistence on the {@link Auth} instance.
+   * Changes the type of persistence on the `Auth` instance.
```

Comment:
revert?

Reply:
This is inside the `Auth` interface so it would just link to itself which would be weird navigation.